cypress/e2e/expenses/Add.cy.js

```diff
@@ -0,0 +1,105 @@
+import '@testing-library/cypress/add-commands';
+
+import {
+  getUKFormattedEuros,
+  getUKFormattedDate,
+  getCurrentMonth,
+} from '../../../src/utils/helpers';
+
+describe('ADD EXPENSE - NOT CURRENT MONTH', () => {
+  before(() => {
+    cy.intercept('GET', '**/items*', []).as('getExpenses');
+    cy.visit('/');
+    cy.wait('@getExpenses');
+  });
+
+  beforeEach(() => {
+    cy.intercept('POST', '**/items*', { fixture: 'expense-post' }).as(
+      'postExpense'
+    );
+  });
+
+  it('adds the expense', () => {
+    cy.addExpenseFromFixture('expense-raw');
+    cy.wait('@postExpense');
+  });
+
+  it('selects expense month', () => {
+    cy.fixture('expense-raw').then(expense => {
+      const expenseMonth = getUKFormattedDate(expense.date, {
+        year: 'numeric',
+        month: 'long',
+      });
+      cy.get(`[id="New expense added to ${expenseMonth}"]`).within(() => {
+        cy.get('[data-cy="okButton"]').click();
+      });
+      cy.findByRole('combobox', { name: /select a month/i }).should(
+        'have.value',
+        expenseMonth
+      );
+    });
+  });
+
+  it('it closes the alert', () => {
+    cy.findByRole('alert').within(() => {
+      cy.get('[class="btn-close"]').click();
+    });
+  });
+
+  it('checks that the form has been cleared', () => {
+    cy.findByRole('textbox', { name: /expense name/i }).should(
+      'have.value',
+      ''
+    );
+    cy.findByLabelText(/expense date/i).should('have.value', '');
+    cy.findByRole('spinbutton', { name: /expense amount/i }).should(
+      'have.value',
+      ''
+    );
+  });
+
+  it('checks that new expense is correctly rendered', () => {
+    cy.get('[data-cy=expenses]')
+      .should('have.length', 1)
+      .and('not.contain', 'You have no prior expenses');
+
+    cy.fixture('expense-post').then(expense => {
+      cy.get('[data-cy="expenses"] > li')
+        .should('contain', getUKFormattedDate(expense.date))
+        .and('contain', `${expense.title}`)
+        .and('contain', `- ${getUKFormattedEuros(expense.amount)}`);
+    });
+  });
+
+  it('checks that the summary has been correctly updated', () => {
+    cy.fixture('expense-post').then(expense => {
```

Comment:
Yep it is possible and it is quite easy, we just import it:
`import expensePost from '../../fixtures/expense-post.json';`

`cy.fixture()` should be used if you want to keep the scope of the fixture you're calling within the `.then()` callback. Thus if we are using that fixture in only one block, we should follow this praxis. Otherwise the normal import makes more sense, yes .

In this case I would import the fixture and use it in the 2 blocks.

Reply:
I will change to import also in any case where we are using the same fixture many times